scripts/run-iroha-dev.sh

```diff
@@ -12,14 +12,10 @@ export COMPOSE_PROJECT_NAME=${PROJECT}
 if [ ! "$(docker ps -q -f name=${PROJECT}_node_1)" ] # node already running
 then
   docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml up -d
-  docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml exec node \
-    adduser --disabled-password --gecos "" ${USER} --uid ${USERID}
-  docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml exec node \
-    chown -R ${USER}:${USER} /tmp/ccache
   docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml exec \
-    --user ${USER} node /bin/bash
+    node /bin/bash
   docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml down
 else
   docker-compose -f ${IROHA_HOME}/docker/docker-compose.yml exec \
-    --user ${USER} node /bin/bash
```

Comment:
these lines are needed because if you build iroha locally, then switch to docker environment, you will have troubles with rights